### ISSUE TITLE
while i use the script `./dev/package.sh` to build gluten, it threw the error: "Architecture not supported!"

### ISSUE THREAD

**Issue by kevincmchen** [apache/incubator-gluten#4299]
### Problem description

while i use the script `./dev/package.sh` to build gluten, it threw the error: "Architecture not supported!", and the following is related codes:

https://github.com/oap-project/velox/blob/5460ec331a2e6841b049ff9b4482cb6f69cf8d80/scripts/setup-helper-functions.sh#L117

after i debugged, i find the cause of the error was due to the following code: 
https://github.com/oap-project/gluten/blob/d1d976baca6e08799068cd99b4ed21b3e6494a4d/dev/package.sh#L10C3-L10C3

in my env, the value of ARCH is `x86_64`, and this value will be used in the following code: 
https://github.com/oap-project/velox/blob/5460ec331a2e6841b049ff9b4482cb6f69cf8d80/scripts/setup-helper-functions.sh#L99

finally it the error: "Architecture not supported!".

### System information

os : ubuntu 20.04
code: gluten branch-1.1

Velox System Info v0.0.2
Commit: cce55960a78c7c3b654b3ac0b2a808791bc6c6ee
CMake Version: 3.16.3
System: Linux-5.4.203-1-tlinux4-0011
Arch: x86_64
C++ Compiler: /usr/bin/c++
C++ Compiler Version: 9.4.0
C Compiler: /usr/bin/cc
C Compiler Version: 9.4.0
CMake Prefix Path: /usr/local;/usr;/;/usr;/usr/local;/usr/X11R6;/usr/pkg;/opt

### CMake log

```bash
+ CPU_TARGET=x86_64
++ get_cxx_flags x86_64
++ local CPU_ARCH=x86_64
++ local OS
+++ uname
++ OS=Linux
++ local MACHINE
+++ uname -m
++ MACHINE=x86_64
++ ADDITIONAL_FLAGS=
++ [[ -z x86_64 ]]
++ [[ x86_64 == \u\n\k\n\o\w\n ]]
++ case $CPU_ARCH in
++ echo -n 'Architecture not supported!'
+ COMPILER_FLAGS='Architecture not supported!'
+ cmake -Wno-dev -B_build -GNinja -DCMAKE_POSITION_INDEPENDENT_CODE=ON -DCMAKE_CXX_STANDARD=17 '' '' '-DCMAKE_CXX_FLAGS=Architecture not supported!' -DBUILD_TESTING=OFF -DFMT_TEST=OFF
-- CMake version: 3.16.3
-- The CXX compiler identification is GNU 9.4.0
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- broken
CMake Error at /usr/share/cmake-3.16/Modules/CMakeTestCXXCompiler.cmake:53 (message):
  The C++ compiler

    "/usr/bin/c++"

  is not able to compile a simple test program.

  It fails with the following output:

    Change Dir: /home/tlinux/codes/opensource/kevin/gluten/ep/build-velox/build/velox_ep/fmt/_build/CMakeFiles/CMakeTmp
    
    Run Build Command(s):/usr/bin/ninja cmTC_fdc90 && [1/2] Building CXX object CMakeFiles/cmTC_fdc90.dir/testCXXCompiler.cxx.o
    FAILED: CMakeFiles/cmTC_fdc90.dir/testCXXCompiler.cxx.o 
    /usr/bin/c++    Architecture not supported!  -fPIE   -std=gnu++17 -o CMakeFiles/cmTC_fdc90.dir/testCXXCompiler.cxx.o -c testCXXCompiler.cxx
    c++: error: Architecture: No such file or directory
    c++: error: not: No such file or directory
    c++: error: supported!: No such file or directory
    ninja: build stopped: subcommand failed.
    
    

  

  CMake will not be able to correctly generate this project.
Call Stack (most recent call first):
  CMakeLists.txt:66 (project)


-- Configuring incomplete, errors occurred!
See also "/home/tlinux/codes/opensource/kevin/gluten/ep/build-velox/build/velox_ep/fmt/_build/CMakeFiles/CMakeOutput.log".
See also "/home/tlinux/codes/opensource/kevin/gluten/ep/build-velox/build/velox_ep/fmt/_build/CMakeFiles/CMakeError.log".

real    0m3.556s
user    0m2.558s
sys     0m0.370s
```

**Comment by gaoyangxiaozhu** [comment]
I think this issue can be closed @PHILO-HE

**Comment by PHILO-HE** [comment]
@kevincmchen, do we still need the created fix? There is no more release in 1.1 branch.

**Comment by kevincmchen** [comment]
> @kevincmchen, do we still need the created fix? There is no more release in 1.1 branch.

this fix is only valid in 1.1.  if there is no more release,  please close this issue.

**Comment by PHILO-HE** [comment]
@kevincmchen, thanks for your feedback. Closing it as no more release in branch 1.1.
